openprescribing/templates/about.html

```diff
@@ -37,7 +37,7 @@ <h3>Code and contributions</h3>
 
 <h3 id="sources">Data sources</h3>
 
-<p><strong>Prescribing data</strong> is from the monthly files published by <a href="https://digital.nhs.uk/practice-level-prescribing-summary">NHS Digital</a>, used under the terms of the Open Government Licence.</p>
+<p><strong>Prescribing data</strong> is from the monthly files published by <a href="https://www.nhsbsa.nhs.uk/information-services-portal-isp">NHS Business Service Authority</a>, used under the terms of the Open Government Licence.</p>
```

Comment:
Should it be "by NHS Business Service Authority" or "by the NHS Business Service Authority"?

Reply:
Are you sure you aren't a pharmacist?!

"by the NHS Business Service Authority"

Can you push it back and I'll change it?

Reply:
> Are you sure you aren't a pharmacist?!

High praise!

You should be able to make this change yourself, without me doing anything.  Let me know if you're not sure how to do this.